augur/util_support/date_disambiguator.py

```diff
@@ -149,3 +142,24 @@ def assert_only_less_significant_uncertainty(self):
                 raise InvalidDate(self.uncertain_date,
                     "Month contains uncertainty, so day must also be uncertain."
                 )
+
+    def set_bounds(self, min_max_year):
+        """Set the exact day bounds."""
+        if len(min_max_year) > 2:
+            raise AugurError(f"The year bounds '{min_max_year}' must have only one (lower) or two (lower, upper) bounds.")
```

Comment:
Use `repr()` formatted values will correctly escape embedded quotes or other unexpected characters. I'd suggest this for all values embedded in messages where it's useful to be very precise (e.g. as indicated here by the manual delimiting by single quotes).

```suggestion
            raise AugurError(f"The year bounds {min_max_year!r} must have only one (lower) or two (lower, upper) bounds.")
```

Reply:
Good idea. Done: 95b7ddd5f80e11a93c806747312385bc635466b9 and also #1147